src/bindings/pyrpr/src/pyrpr.py

```diff
@@ -561,14 +561,23 @@ def mark_static(self, is_static):
     def set_vertex_value(self, index: int, indices, values):
         ShapeSetVertexValue(self, index, ffi.cast("rpr_int *", indices.ctypes.data),
                             ffi.cast("float *", values.ctypes.data), len(indices))
-
+    
+    def set_primvar(self, key: int, data, floatcount: int, count: int, interop):
```

Comment:
rename some variables: 
float_count
interop -> interp or interp_type

Reply:
will do, was mostly following the names from the sdk. I thought interop was weird as well but didn't want to use and other term that was used somewhere else (which is why I figured they did it)